integration_tests/integration/serviceProviderReferrals.spec.js

```diff
@@ -1094,7 +1094,7 @@ describe('Service provider referrals dashboard', () => {
 
     describe('with valid inputs and an appointment in the past', () => {
       describe('when booking for a phone call', () => {
-        it('should present no errors and display scheduled appointment', () => {
+        it('should present a form validation error', () => {
```

Comment:
This reads funny

with valid inputs and an appointment in the past
when booking for a phone call
it should present a form validation error

Can we change the precondition statements to reflect why it should fail?